history.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: page
+title: History
+slug: history
+
+---
+
+## One Laptop Per Child
+
+One Laptop Per Child ([OLPC](https://laptop.org/)) was proposed by MIT in 2005 to [fight the gap](https://laptop.org/aboutolpc/) in access to technology in education for underprivileged children.
+
+Stephen Jacobs, director of RIT's Lab for Technological Literacy (LTL) at the time, was interested in the program and acquired three OLPC XO 1.0 laptops in 2007 via the first "Give One, Get One" program. 
+In January 2009, Rochester Institute of Technology had an honors class that developed educational games for the OLPC program. That was the earliest documented birth of Open-Source @ RIT. 
+Open-source advocates, such as Karlie Robinson and David Nalley, were brought in by Professor Jacobs to support the class. 
+Karlie Robinson introduced the students to the Sugar Labs' Math 4 initiative which was dedicated to supporting 4th grade curricula. 
+David Nalley, who was part of the Fedora community, donated 25 XO laptops to help students contribute to the Sugar initiative.
+This ended the first iteration of the class, but the seminar class went on for several more iterations, with more incredible open-source advocates joining the picture.
+Sadly, the original history was based on an old [magic.rit.edu](https://web.archive.org/web/20150427134219/http://magic.rit.edu/foss/history.html) page, now it no longer exists.
+
+
+
+## FOSSBOX:
+
+Estimated to have started in 2010, Stephen Jacobs hired Remy DeCausemaker (RIT alumni) to run the open-source program. Remy was initially from the Red Hat Corporation so the club had a great foundation.
```

Comment:
```suggestion
Estimated to have started in 2010, Stephen Jacobs hired Remy DeCausemaker (RIT alumni) to run the Open Source program.
Remy was initially from the Fedora community so the club had a great foundation.
```

I am pretty sure @decause's chapter at Red Hat started well after his RIT chapter, but he was involved with Fedora from some time before Red Hat and I think before RIT too. I could be wrong, maybe he can help set the record straight here. 🙂